velero-plugins/common/types.go

```diff
@@ -23,3 +23,12 @@ const RestoreRegistryHostname string = "openshift.io/restore-registry-hostname"
 const MigrationRegistry string = "openshift.io/migration-registry"
 
 const SwingPVAnnotation string = "openshift.io/swing-pv"
+
+// copy, swing, TODO: others (snapshot, custom, etc.)
+const MigrateTypeAnnotation string = "openshift.io/migrate-type"
+
+//stage, migrate-phase1, migrate-final. Only valid for copy type.
+const MigrateCopyPhaseAnnotation string = "openshift.io/migrate-copy-phase"
```

Comment:
+1 much prefer specifying phase in an annotation than how we were doing this previously